src/core/public/rendering/rendering_service.tsx

```diff
@@ -65,12 +66,12 @@ export class RenderingService {
           {chromeUi}
 
           {!legacyMode && (
-            <div className="app-wrapper">
+            <AppWrapper chromeVisible$={chrome.getIsVisible$()}>
               <div className="app-wrapper-panel">
                 <div id="globalBannerList">{bannerUi}</div>
-                <div className="application">{appUi}</div>
+                <AppContainer classes$={chrome.getApplicationClasses$()}>{appUi}</AppContainer>
```

Comment:
I fixed this in the same PR as we missed it ( the chrome applicationClasses were only properly used when rendering legacy apps), however I did not add functional tests for this, as I'm not sure what the actual usage should be. Tell me if unit test are not enough.